android/src/main/java/com/frontegg/android/AuthenticationActivity.kt

```diff
@@ -104,9 +104,9 @@ class AuthenticationActivity : Activity() {
         private const val CUSTOM_TAB_LAUNCHED = "com.frontegg.android.CUSTOM_TAB_LAUNCHED"
         private val TAG = AuthenticationActivity::class.java.simpleName
 
-        fun authenticate(activity: Activity) {
+        fun authenticate(activity: Activity, loginHint: String?) {
```

Comment:
I prefer to create a new method called authenticateWithLoginHint to prevent breaking changes.
or use default value to loginHint
```
fun authenticate(activity: Activity, loginHint: String? = null)
```

Reply:
Added default value to loginHint